exotica_python/src/pyexotica.cpp

```diff
@@ -742,7 +742,8 @@ PYBIND11_MODULE(_pyexotica, module)
         .def("reset_number_of_problem_updates", &PlanningProblem::ResetNumberOfProblemUpdates)
         .def("get_cost_evolution", (std::pair<std::vector<double>, std::vector<double>>(PlanningProblem::*)() const) & PlanningProblem::GetCostEvolution)
         .def("get_number_of_iterations", &PlanningProblem::GetNumberOfIterations)
-        .def("is_valid", &PlanningProblem::IsValid);
+        .def("is_valid", &PlanningProblem::IsValid)
+        .def_readonly("termination_criterion", &PlanningProblem::termination_criterion);
```

Comment:
def_property_readonly

Reply:
But it's not a property? It's just a public member.